server/main.lua

```diff
@@ -39,26 +39,23 @@ end)
 
 ---@param player table|number
 local function revivePlayer(player)
-	if type(player) == 'number' then
-		player = exports.qbx_core:GetPlayer(player)
-	end
-	TriggerClientEvent('qbx_medical:client:playerRevived', player.PlayerData.source)
+    TriggerClientEvent('qbx_medical:client:playerRevived', player --[[@as number]])
 end
 
 exports('Revive', revivePlayer)
 
 ---removes all ailments, sets to full health, and fills up hunger and thirst.
 ---@param src Source
 local function heal(src)
-	lib.callback.await('qbx_medical:client:heal', src, 'full')
+    TriggerClientEvent('qbx_medical:client:heal', src, 'partial')
```

Comment:
Why the change from a full to a partial heal?

Reply:
just a typo